packages/sg1/src/lib.rs

```diff
@@ -102,12 +103,12 @@ mod tests {
         fair_burn(1000u128, Some(Addr::unchecked("geordi")), &mut res);
         let bank_msg = SubMsg::new(BankMsg::Send {
             to_address: "geordi".to_string(),
-            amount: coins(100, "ustars".to_string()),
+            amount: coins(500, NATIVE_DENOM),
```

Comment:
Should we have a test where the amount for rewards is not an even number (to test the decimal placement)?